<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="6" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="49" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="49" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
      <class name="mavendemo.day1.firstTest">
        <test-method status="FAIL" signature="aTest()[pri:0, instance:mavendemo.day1.firstTest@57fffcd7]" name="aTest" duration-ms="5" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at mavendemo.day1.firstTest.aTest(firstTest.java:10)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aTest -->
        <test-method status="PASS" signature="bTest()[pri:0, instance:mavendemo.day1.firstTest@57fffcd7]" name="bTest" duration-ms="0" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bTest -->
      </class> <!-- mavendemo.day1.firstTest -->
      <class name="mavendemo.day1.secondTest">
        <test-method status="PASS" signature="aTest()[pri:0, instance:mavendemo.day1.secondTest@59ec2012]" name="aTest" duration-ms="0" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aTest -->
        <test-method status="PASS" signature="bTest()[pri:0, instance:mavendemo.day1.secondTest@59ec2012]" name="bTest" duration-ms="0" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bTest -->
      </class> <!-- mavendemo.day1.secondTest -->
      <class name="mavendemo.day1.thirdTest">
        <test-method status="PASS" signature="aTest()[pri:0, instance:mavendemo.day1.thirdTest@4cf777e8]" name="aTest" duration-ms="0" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aTest -->
        <test-method status="PASS" signature="bTest()[pri:0, instance:mavendemo.day1.thirdTest@4cf777e8]" name="bTest" duration-ms="0" started-at="2022-07-15T01:35:37 IST" finished-at="2022-07-15T01:35:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bTest -->
      </class> <!-- mavendemo.day1.thirdTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
